extension/src/init.js

```diff
@@ -0,0 +1,8 @@
+const init = require('./server.js')
+require('./config/config')
+
+const port = 8080
```

Comment:
It can be an env variable

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/4/commits/71a1028fa7b6a8ac7f47f45c09c18b2d92a6dc53